src/app/course/models/course.ts

```diff
@@ -0,0 +1,75 @@
+import {FormControl} from "@angular/forms";
+
+export type TCourse = {} & {
+	id?: number;
```

Comment:
To make all fields optional you can simpy use `Partial<Cource>`
`{} ` looks useless here

Reply:
begin use Partial